src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -214,14 +214,15 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
             HubMethodInvocationMessage hubMethodInvocationMessage, bool isStreamResponse, bool isStreamCall)
         {
             var methodExecutor = descriptor.MethodExecutor;
+            var hubName = typeof(THub).Name;
```

Comment:
nit: move this into the `IsHubMethodAuthorized` method after the `policies.Any()` check